web/ui/react-app/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function(app) {
+  app.use(
+    '/api',
+    createProxyMiddleware({
+      target: 'http://localhost:9090',
+      changeOrigin: true,
+    })
+  );
+};
```

Comment:
GitHub is warning about the lack of a newline at the end of the file here, could you add that?

Reply:
ok